api/import.go

```diff
@@ -19,10 +20,24 @@ import (
 //
 
 func ImportPost(post *model.Post) {
-	post.Hashtags, _ = model.ParseHashtags(post.Message)
+	for utf8.RuneCountInString(post.Message) > 0 {
+		var remainder string
+		if utf8.RuneCountInString(post.Message) > model.POST_MESSAGE_MAX_RUNES {
```

Comment:
Oh hang on, I think the code I just pushed changing this doesn't do what I meant it to. Let me just check.